### PR TITLE
[jarvis] Added launch capabilities to jarvis

### PR DESCRIPTION
**Pull request by mmartin4972**
Added launch capabilities to jarvis. Basically you type ./jarvis launch <package> and it will build, open a terminal with, and then run your code. Can and has been applied to multiple packages. Launch scripts for loc and nav scripts need revision. 

This PR was integration tested in PR #663

### CODE REVIEW

**Comment by mmartin4972** [comment]
This PR was integration tested in PR #663

**Review by zwgold — DISMISSED** [review]
Add note that dependency for this needs to be added to `base_station` deps.

**Review by niwhsa9 — DISMISSED** [review]
neat, LGTM

File: jarvis_files/launch_scripts/loc [hunk 1]
```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# Focus in on the current window
+WID=$(xdotool selectwindow)
+xdotool windowfocus $WID
+
+# Enter commands you want to be executed in the window here
+
+# Run Perception
```
[the remark this answers is not in the document]
Comment (mmartin4972) [reply jarvis_files/launch_scripts/loc]: Fixed

File: jarvis_files/launch_scripts/nav [hunk 2]
```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Focus in on the current window
+WID=$(xdotool selectwindow)
+xdotool windowfocus $WID
+
+# Enter commands you want to be executed in the window here
+
+# Run Perception
```
[the remark this answers is not in the document]
Comment (mmartin4972) [reply jarvis_files/launch_scripts/nav]: Fixed. Next time you want changes reject the PR and require me to make the changes. Then approve the PR. Also, please click the 'Resolve conversation' once the issue was resolved

**Comment by mmartin4972** [comment]
Specified the addition of the "xdotool" dependency in Issue #705

**Review by cgiger00 — DISMISSED** [review]

**Comment by mmartin4972** [comment]
> Specified the addition of the "xdotool" dependency in Issue #705

Added the dependency so can close this issue

**Comment by mmartin4972** [comment]
Needs to be integration tested first.

**Pull request closed without merging**